build_msvc/msvc-autogen.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import os
+import re
+
+SOURCE_DIR=os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'src'))
```

Comment:
Nit: Missing space around `=`.

The `flake8` rule set enforced by Travis is a bit lax due to historical reasons, but since this is brand new code I suggest following all the advice given by `flake8` to avoid any Python style nits :-)

```
$ flake8 --ignore=E501 msvc-autogen.py # E501 is line length
msvc-autogen.py:6:11: E225 missing whitespace around operator
msvc-autogen.py:27:1: E302 expected 2 blank lines, found 1
msvc-autogen.py:29:20: E225 missing whitespace around operator
msvc-autogen.py:33:79: E713 test for membership should be 'not in'
msvc-autogen.py:36:32: E225 missing whitespace around operator
msvc-autogen.py:39:39: E231 missing whitespace after ','
msvc-autogen.py:41:32: E225 missing whitespace around operator
msvc-autogen.py:45:1: E302 expected 2 blank lines, found 1
msvc-autogen.py:58:1: E305 expected 2 blank lines after class or function definition, found 1
```

Reply:
Fixed